module5/Publisher.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+from typing import List
+
+from module5.ArabicText import ArabicText
+from module5.InputException import InputException
+from module5.News import News
+from module5.PrivateAd import PrivateAd
+from module5.Publication import Publication
+
+
+class Publisher:
+
+    @staticmethod
+    def get_publication_type() -> str:
+        pub_type = input("Welcome to Publisher!\n"
```

Comment:
Possibly, let user choose from simple list like 1,2,3 instead of input text where misspelling could happen

Reply:
Fixed by enum.